app/api/v1/openai.py

```
@@ -28,28 +31,52 @@ async def openai_chat_completions(
+        if stream:
+            stream_response = cast(CustomStreamWrapper, llm_response)
+            return StreamingResponse(
+                generate_stream_response(stream_response),
+                media_type="text/event-stream",
+            )
+        else:
+            return llm_response
     except Exception as e:
         raise HTTPException(
             status_code=500, detail=f"Error processing request: {str(e)}"
         )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)